### PR TITLE
[ATH-1171] View Single Blog Article - Rails Implementation

### PR DESCRIPTION
**Pull request by meganduffy**
View full blog articles in their own page. 

---

**To view content:**

Run the rails server and navigate to the `/article` URL, or by clicking on the View Articles link on the homepage.

Once there, click on the View Full Article link beneith any article in order to view the full thing. 

**To run tests:**

```
rspec

yarn test
```

__Please note the next story will be focused on the view implementation.__

### CODE REVIEW

File: spec/controllers/articles_controller_spec.rb [hunk 1]
```diff
@@ -145,4 +145,26 @@
...
     end
   end
+
+  describe 'GET #show' do
+    context 'when user is logged in' do
+      before(:each) do
+        allow(controller).to receive(:current_user).and_return(user)
+      end
+
+      it 'returns a 200 OK HTTP response' do
```
Comment (jamesfairbairn-wpengine) [inline spec/controllers/articles_controller_spec.rb:155]: Might be good to add a test that verifies it's displaying the right information as well.

Comment (meganduffy) [reply spec/controllers/articles_controller_spec.rb:155]: Great idea. I was going to leave that out as the next story will change the tests but after thinking about it I realise that each story should be independent of all others and should have comprehensive testing - even if those tests are expected to change.

Comment (alvarocavalcanti) [reply spec/controllers/articles_controller_spec.rb:155]: That's right! I like to see things as this: consider that every code you push to production will potentially live unmodified forever. You never know what the future holds, and that improvement you have in mind will never get prioritized. Thus, make sure your code is the best shape possible.

Comment (meganduffy) [reply spec/controllers/articles_controller_spec.rb:155]: It's little learning curves like this that I'm so excited to be experiencing before jumping into sprint work! Acceleration project FTW! 🙌

**Pull request merged**